thegame/engine/game_objects/character_object.py

```diff
@@ -78,15 +78,19 @@ def move(self, context, up=False, down=False, left=False, right=False):
 
         if up:
             new_pos = (current_pos[0], current_pos[1] - 1)
```

Comment:
@Ross-Alexandra Umm, it doesn't compile when I do that...

Reply:
Are you using "" as the actual sprite (rather than the ones you were using here? As long as GameObject has a set_sprite method, PlayerControlledObject should....

```
>>> class Foo:
...     def __init__(self):
...             self.a = 5
...     def double(self):
...             self.a *= 2
...
>>>
>>> class Bar(Foo):
...     def triple(self):
...             self.a *= 3
...
>>>
>>> myobj = Bar()
>>> myobj.a
5
>>> myobj.double
<bound method Foo.double of <__main__.Bar object at 0x7feb7231deb8>>
>>> myobj.double()
>>> myobj.a
10
>>> myobj.triple()
>>> myobj.a
30
```

Reply:
Okay, `self.set_sprite()` works. Honestly I thought I tried that before and it didn't work but whatever lol.